exporter/coralogixexporter/config.go

```diff
@@ -56,7 +58,7 @@ func (c *Config) Validate() error {
 		return fmt.Errorf("`appName` not specified, please fix the configuration file")
 	}
 	if c.SubSystem == "" {
-		return fmt.Errorf("`subSystem` not specified, please fix the configuration file")
+		return fmt.Errorf("`subSystem` not specified, please fix the configuration file.\nPay attention: deprecated subsystem will be removed in the next version")
```

Comment:
Imagine error messages in a broader context: they bubble up and are printed to the user by the component that is the closest to the user, collecting all the breadcrumbs until it gets there. In that context, new lines aren't desirable, as your message would end up looking like this:

    2022-03-11 14:10:12.1234 ERR could not start blurbs: failed to parse config file: `subSystem` not specified, please fix the configuration file
    Pay attention: deprecated subsystem will be removed in the next version.

That said, you should really make room for your users to migrate to the replacement already. If they won't be allowed to specify this option, then make it possible for your users to not specify this already and leave their config files ready for the next version.

Reply:
thanks @jpkrohling , I fixed the error message and let the users add or remove 'subsystem' option from the config file in this version.